docs/sources/whatsnew/whats-new-next/index.md

```diff
@@ -118,6 +118,19 @@ To learn more, refer to the following video demo.
 
 {{< video-embed src="/media/docs/grafana/2023-09-11-New-Browse-Dashboards-Enablement-Video.mp4" >}}
 
+## Multiple spansets per trace
+
+<!-- Joey Tawadrous -->
+<!-- OSS, Enterprise -->
+
+October 24, 2023
+
+_Generally available in Grafana Cloud_
+
+The [TraceQL query editor](https://grafana.com/docs/tempo/<TEMPO VERSION>/traceql/#traceql-query-editor) has been improved to facilitate the grouping of multiple spansets per trace in TraceQL queries. For example, when the following `by(resource.service.name)` is added to your TraceQL query, it will group the spans in each trace by `resource.service.name`.
```

Comment:
@jdbaldry, When i build this locally, the version interpolation syntax seems to break the link. Should I just go with "latest"?
```suggestion
The [TraceQL query editor](https://grafana.com/docs/tempo/<TEMPO VERSION>/traceql/#traceql-query-editor) has been improved to facilitate the grouping of multiple spansets per trace in TraceQL queries. For example, when the following `by(resource.service.name)` is added to your TraceQL query, it will group the spans in each trace by `resource.service.name`.
```

Reply:
`TEMPO VERSION` is not yet defined in Grafana Cloud.

I've added it in https://github.com/grafana/website/pull/15879/files.

By the way, did you notice that website builds were faster or no?

Reply:
Unfortunately, no, the build wasn't any faster for me. :-(